Dockerfile

```diff
@@ -9,6 +9,8 @@ ARG DUCKDNS_VERSION
 LABEL build_version="Linuxserver.io version:- ${VERSION} Build-date:- ${BUILD_DATE}"
 LABEL maintainer="aptalca"
 
+ENV UPDATE_IP=ipv4
```

Comment:
I wouldn't recommend this. Portainer and synology gui read this and externally set it on an updated container: https://info.linuxserver.io/issues/2023-05-22-portainer/

It's better to modify the if block in the init script so the `ipv6` and `both` can be the first two tests (`if`/`elif`), and `else` would do ipv4 only. That way ipv4 is truly the default option, so even when the user sets it to something invalid, the code will treat it as `ipv4`.

Reply:
Right. I did not know that... I fix